apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -216,27 +220,21 @@ GoogleBlockly.Extensions.registerMixin(
  * @param {WorkspaceSvg} workspace The workspace containing procedures.
  * @returns an array of block objects representing the flyout blocks
  */
-export function flyoutCategory(workspace, functionEditorOpen = false) {
+export function flyoutCategory(workspace) {
   const blockList = [];
 
-  // Note: Blockly.Msg was undefined when this code was extracted into global scope
-  const functionDefinitionBlock = {
```

Comment:
We only need to define this block if we aren't using the modal editor (so that we can add it directly to the flyout blocks).